src/order.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default.html
+---
+
+<h2>Order</h2>
+<p>Please use this to order biscuits!</p>
+<form action="/action_page.php">
+    <fieldset>
+      <legend>Order form:</legend>
+      <label for="name">Name:</label><br>
+      <input type="text" id="name" name="name" value="John"><br>
+      <label for="email">Email:</label><br>
+      <input type="text" id="email" name="email" value="john@email.com"><br>
```

Comment:
There's a separate `type` of `input` for email addresses:
```suggestion
      <input type="email" id="email" name="email" value="john@email.com"><br>
```
Learn more about valid `type` attributes for the `input` element here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types